command.go

```diff
@@ -1050,7 +1070,7 @@ func (c *Command) ExecuteC() (cmd *Command, err error) {
 			c = cmd
 		}
 		if !c.SilenceErrors {
-			c.PrintErrln("Error:", err.Error())
+			c.PrintErrln(cmd.ErrPrefix(), err.Error())
```

Comment:
This line should have stayed with `c.ErrPrefix()`.  Notice that right above, `c = cmd` as long as `cmd != nil`.  So, in case `cmd == nil` we should use `c` here.

Reply:
Oops, thank you for describing!